src/qibo/backends/clifford.py

```diff
@@ -150,6 +172,28 @@ def execute_circuit(self, circuit, initial_state=None, nshots: int = 1000):
         Returns:
             :class:`qibo.quantum_info.clifford.Clifford`: Object storing to the final results.
         """
+        from qibo.quantum_info.clifford import Clifford  # pylint: disable=C0415
+
+        if self.platform == "stim":
+            circuit_stim = self._stim.Circuit()  # pylint: disable=E1101
+            for gate in circuit.queue:
+                circuit_stim.append(gate.__class__.__name__, list(gate.qubits))
+
+            quad_1, quad_2, quad_3, quad_4, x_phases, z_phases = (
+                self._stim.Tableau.from_circuit(  # pylint: disable=no-member
+                    circuit_stim
+                ).to_numpy()
+            )
+            symplectic_matrix = np.block([[quad_1, quad_2], [quad_3, quad_4]])
+            symplectic_matrix = np.c_[symplectic_matrix, np.r_[x_phases, z_phases]]
```

Comment:
In principle, it should be possible to do just:
```suggestion
            symplectic_matrix = np.block([[quad_1, quad_2], [quad_3, quad_4], [x_phases, z_phases]])
```
or similarly (I'm not sure about the role of `quad_x`, to which blocks they map to).

I'd say it is more intuitive.

Reply:
This would concatenate `[x_phases, z_phases]` as the last row, but they are the last column.